src/components/Authentication/Authentication.vue

```diff
@@ -0,0 +1,94 @@
+<template lang="pug">
+v-flex.pa-2.girder-authentication-component(xs-12)
```

Comment:
We should remove this outer element. To make this reusable in a variety of contexts, we'll let callers specify what sort of layout or container it should be in. To that end, I also think we should either make the outer `v-card(flat)`, or remove it altogether so that downstreams can decide for themselves whether they want box shadow / border radius around this element. Another alternative would be to add `flat` as a prop of this component.

As a general rule, we shouldn't add padding or margin to the top level elements of a reusable component, we should only use it to separate elements inside them. We should think of these components as just the "inside" of some containing element, and we should add the containing elements inside our `App.vue` example.